content/applications/services/helpdesk/overview/reports.rst

```diff
@@ -197,25 +180,27 @@ chart, while in graph view.
 
 .. tip::
    Both the *bar chart* and *line chart* can utilize the *stacked* view option. This presents two
-   (or more) groups of data on top of each other, instead of next to each other, making it easier
-   to compare data.
+   or more groups of data on top of each other, instead of next to each other, making it easier to
+   compare data.
 
 Save and share a favorite search
 --------------------------------
 
 The *Favorites* feature found on *Helpdesk* reports allows users to save their most commonly used
 filters without having to reconstruct them every time they are needed.
 
-To create and save new *Favorites* on a report, follow the steps below:
+To create and save new *Favorites* configuration on a report, follow the steps below:
 
 #. Set the necessary parameters using the :guilabel:`Filters`, :guilabel:`Group By` and
    :guilabel:`Measures` options.
-#. Click :menuselection:`Favorites --> Save current search`.
-#. Rename the search.
-#. Select :guilabel:`Use by default` to have these filter settings automatically displayed when the
-   report is opened. Otherwise, leave it blank.
-#. Select :guilabel:`Share with all users` to make this filter available to all other database
-   users. If this box is not checked, it will only be available to the user who creates it.
+#. Click the :guilabel:`🔻(triangle pointed down)` icon next to the :guilabel:`Search...` bar to
+   open the drop-down menu.
+#. Under the :guilabel:`Favorites` heading, click :guilabel:`Save current search`.
+#. If desired, enter a new name for the report.
+#. Tick the :guilabel:`Default Filter` checkbox to have these filter settings automatically
+   displayed when the report is opened. Otherwise, leave it blank.
+#. Tick the  :guilabel:`Shared` checkbox to make this filter configuration available to all other
+   database users. If this box is not checked, only the user who creates the filter can access it.
```

Comment:
This doc is really useful, and it's great that you're fleshing out all of these reports and fields. As a helpdesk beginner though, I'd like to share some of my first impressions.

When looking at a pivot table like the one in the SLA Status Analysis report, I have to admit, I was a bit lost. It might be my lack of industry knowledge, though!

So, maybe in a followup PR, when you get a chance, you could include some example graphs and give us a few pointers on what numbers we should be paying attention to. Like, what's considered a red flag in terms of ticket start/finish times?

Just a thought!

Reply:
Thats a good point, I do talk a bit about this report in the SLA doc (which I will add a link to) but I can update that section to provide some additional metrics guidelines :+1: